farmerbot/internal/find_node_test.go

```diff
@@ -67,65 +67,71 @@ func TestFindNode(t *testing.T) {
 	t.Run("test valid find node: found an ON node", func(t *testing.T) {
 		nodeID, err := farmerbot.findNode(sub, nodeOptions)
 		assert.NoError(t, err)
-		assert.Contains(t, farmerbot.nodes, nodeID)
 
-		farmerbot.addNode(oldNode1)
-		farmerbot.addNode(oldNode2)
+		_, node, err := farmerbot.getNode(nodeID)
+		assert.NoError(t, err)
+		assert.Contains(t, farmerbot.nodes, node)
+
+		assert.NoError(t, farmerbot.updateNode(oldNode1))
+		assert.NoError(t, farmerbot.updateNode(oldNode2))
```

Comment:
Why are we updating the `oldNodes`

Reply:
To reset changes made by `findNode` function